app/src/main/res/drawable/vd_thumbs_down.xml

```diff
@@ -0,0 +1,4 @@
+<vector android:height="18dp" android:viewportHeight="24"
```

Comment:
We prefer to name vector drawable using this convention, `vd_` followed by the material drawable name. In this case it will be `vd_thumb_up_outline`

Reply:
I switched both of the thumbs to the correct naming convention. I also switched to the outline vector as the one before was filled.